foundation-extensions/profile/README.md

```diff
@@ -175,25 +213,24 @@ public class MobileApp extends Application {
 }
 ```
 
-### Swift
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+#### Swift
```

Comment:
All the headers should use the same level and I believe h4 shows the same as bold, so h3 might be more appropriate. @calebk1 can you please confirm if we should use h3/h4 for the programming language header?